src/components/ebay-video/marko-tag.json

```diff
@@ -12,8 +12,11 @@
   "@cdn-version": "string",
   "@a11y-play-text": "string",
   "@a11y-load-text": "string",
+  "@a11y-report-menu": "string",
```

Comment:
What's the difference between `a11y-report-menu` and `report-text`?

Reply:
I missed changing this, good catch!
Changed it to `a11y-report-text`